docs/infoblox-openshift.md

```diff
@@ -91,7 +92,8 @@ Make sure your AWS account is subscribed to the following product:
     aws cloudformation describe-stacks --stack-name infoblox
     ```
     _Note_: NIOS instance takes around 10 minutes to start
-- Once the EC2 instance passed all the checks you can try to connect to the Grid Manager WebUI using the Elastic IP and `admin/${GRID_ADMIN_PASSWORD}` as credentials. Use HTTPS scheme and accept the self signed certificate
+- Once the EC2 instance passed all the checks you can try to connect to the Grid Manager WebUI using the Elastic IP and `admin/${GRID_ADMIN_PASSWORD}` as credentials. Use HTTPS scheme and accept the self signed certificate.
+- You may want to set up the DNS for the newly created public IP. You will need the FQDN later during the setup.
```

Comment:
*nit* This might just be how I speak english, but "the DNS" sounds weird to me.

```suggestion
- You may want to set up DNS for the newly created public IP. You will need the FQDN later during the setup.
```